tutor/templates/apps/openedx/settings/partials/common_lms.py

```diff
@@ -38,9 +38,9 @@
 }
 
 # Create folders if necessary
-for folder in [DATA_DIR, LOG_DIR, MEDIA_ROOT, STATIC_ROOT_BASE, ORA2_FILEUPLOAD_ROOT]:
+for folder in [DATA_DIR, LOG_DIR, MEDIA_ROOT, STATIC_ROOT, ORA2_FILEUPLOAD_ROOT]:
     if not os.path.exists(folder):
-        os.makedirs(folder, exist_ok=True)
+        os.makedirs(folder, exist_ok=True, parents=True)
```

Comment:
Note for reviewers: `parents=True` here is a no-op, but I added it just for consistency with the [corresponding CMS block](https://github.com/overhangio/tutor/pull/1042/files#r1592817573).